pyproject.toml

```diff
@@ -1,41 +1,44 @@
 [build-system]
-requires = ["flit_core >=3.2,<4"]
-build-backend = "flit_core.buildapi"
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
 
-[project]
+[tool.poetry]
 name = "ch-tools"
-license = { file = "LICENSE" }
-dynamic = ["version", "description"]
+version = "2.529.131159744"
```

Comment:
Shouldn't it be set dynamically using value from `version.txt`?

And target versioning schema, I suppose, should be based on git tags. Probably with the help of https://github.com/pypa/setuptools_scm. Though this is out of scope of the given PR.

Reply:
Yes it is updated dynamically [here](https://github.com/yandex/ch-tools/blob/3cc61b25704bbc71145e23410749fa61d865813b/Makefile#L279). I accedintally uploaded it after testing bulding deb packages. As for schema and other refinments, I use the current one and it can be changed in the next PRs.